assets/prototype/domain/shared/predicates/datetimes/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const MINUTE_IN_SECONDS = 60;
+export const HOUR_IN_SECONDS = MINUTE_IN_SECONDS * 60;
+export const DAY_IN_SECONDS = HOUR_IN_SECONDS * 24;
+export const WEEK_IN_SECONDS = HOUR_IN_SECONDS * 7;
+export const MONTH_IN_SECONDS = DAY_IN_SECONDS * 30;
```

Comment:
These could be in some common place instead of inside `/datetimes/`?

Reply:
Lifted up, thx: https://github.com/eventespresso/event-espresso-core/pull/2087/commits/54706d55fae9b68e054af265c3d922366000d103